Dockerfile

```diff
@@ -4,9 +4,10 @@ RUN apk -v --update add \
         py-pip \
         groff \
         less \
+        jq \
         mailcap \
         && \
-    pip install --upgrade awscli==1.14.5 s3cmd==2.0.1 python-magic && \
+    pip install --upgrade awscli==1.16.11 s3cmd==2.0.2 python-magic && \
```

Comment:
@brentryan  I suppose it might be easier to get this merged if you only add the new jq line, and not introduce new versions of awscli and s3cmd in the same commit.